applets/brightness/brightness-applet.c

```diff
@@ -408,10 +408,14 @@ gpm_applet_update_popup_level (GpmBrightnessApplet *applet)
  * callback for the plus button
  **/
 static gboolean
-gpm_applet_plus_cb (GtkWidget *w, GpmBrightnessApplet *applet)
+gpm_applet_plus_cb (GtkWidget *w, GpmBrightnessApplet *applet, int value)
 {
 	if (applet->level < 100) {
-		applet->level++;
+		if (applet->level + value > 100) {
+			applet->level = 100;
+		} else {
+			applet->level += value;
+		}
```

Comment:
I would simplify this block a bit
```c
applet->level += value;
if (applet->level > 100)
    applet->level = 100;
```
You potentially do an extra assignment, but you remove a conditional branch, plus it's easier to read (imho)

Reply:
True, I'll do that and pull request again